src/web/app/components/question-types/question-edit-answer-form/rubric-question-edit-answer-form.component.html

```diff
@@ -10,9 +10,26 @@
       <td *ngFor="let rubricDescriptionCell of rubricDescriptionRow; let j = index;" class="text-secondary answer-cell" (click)="selectAnswer(i, j)">
         <label>
           <input type="radio" [disabled]="isDisabled"
-                 [name]="id + i" [checked]="i < responseDetails.answer.length && responseDetails.answer[i] === j"/> {{ rubricDescriptionCell }}
+                 [name]="id + i + '-desktop'" [checked]="i < responseDetails.answer.length && responseDetails.answer[i] === j"/> {{ rubricDescriptionCell }}
         </label>
       </td>
     </tr>
   </tbody>
 </table>
+<div class="mobile-view">
+  <div class="card" *ngFor="let rubricDescriptionRow of questionDetails.rubricDescriptions; let i = index;">
+    <div class="card-header bg-light">
+      {{ questionDetails.rubricSubQuestions[i] }}
+    </div>
+    <div class="card-body"
```

Comment:
We should have `(click)="selectAnswer(i, j)"` here?

Reply:
No. `j` is not defined at this block.

Reply:
The problem is that the model only get updated after `selectAnswer(i, j)` is called.